docs/get-started.md

```diff
@@ -11,13 +11,38 @@ weight=-70
 
 # Get started with Docker Machine and a local VM
 
-Let's take a look at using `docker-machine` for creating, using, and managing a
-Docker host inside of <a href=
-"https://www.virtualbox.org/" target="_blank">VirtualBox</a>.
+Let's take a look at using `docker-machine` to create, use and manage a
+Docker host inside of a local virtual machine.
+
+## Prerequisite Information
+
+With the advent of [Docker for Mac](/docker-for-mac/index.md) and [Docker for Windows](/docker-for-windows/index.md) as replacements for [Docker Toolbox](/toolbox/overview.md), there are some new considerations for creating local machines with `docker-machine create`.
```

Comment:
Nitpick -- I think it's somewhat known that they're intended to be replacements, but I think we might want to come across slightly stronger here, declaring that users are recommended to switch to these for their primary workflow, leading into something like, "However, Machine is still able to create and manage machines for power users or multi-node experimentation, with a few considerations to keep in mind".